packages/amplify-cli/package.json

```diff
@@ -66,7 +66,7 @@
     "@aws-amplify/amplify-util-mock": "5.9.3",
     "@aws-amplify/amplify-util-uibuilder": "1.14.2",
     "@aws-cdk/cloudformation-diff": "~2.68.0",
-    "amplify-codegen": "^4.7.4",
+    "amplify-codegen": "^4.7.3",
```

Comment:
Here and throughout: Make this an explicit version pin rather than an upwards-compatible version. (Remove `^`)